pkg/codegen/schema/loader.go

```diff
@@ -171,9 +182,7 @@ func (l *pluginLoader) LoadPackageReference(pkg string, version *semver.Version)
 	if err != nil {
 		return nil, err
 	}
-	// We assume that GetSchema isn't implemented it something of the form "{[\t\n ]*}" is
-	// returned. That is what we did in the past when we chose not to implement GetSchema.
-	if s := string(schemaBytes); strings.TrimSpace(strings.TrimSuffix(strings.TrimPrefix(s, "{"), "}")) == "" {
```

Comment:
Gotta love casts that copy.

Reply:
The cast should not copy, but research tells me they do... 😒.